Formula/quictls.rb

```diff
@@ -0,0 +1,132 @@
+class Quictls < Formula
+  desc "This is a fork of OpenSSL to enable QUIC"
```

Comment:
```suggestion
  desc "TLS/SSL and crypto library with QUIC APIs"
```
Let's use the description from the repo.